packages/salesforcedx-vscode-core/package.json

```diff
@@ -47,6 +47,7 @@
     "@types/adm-zip": "^0.5.0",
     "@types/chai": "4.3.3",
     "@types/glob": "^7.2.0",
+    "@types/jest": "28.1.7",
```

Comment:
what was weird, is that jest was in vscode-core's `scripts`, and one could run jest against vscode-core, but vscode-core's dependencies didn't include Jest.  I added them to match other projects.